apis/otpapi/otpapi.yaml

```diff
@@ -0,0 +1,407 @@
+openapi: 3.0.1
+info:
+  title: OTP API(s)
+  description: 'The Email resources provides the notification mechanism for all the
+    users on Sunbird. The Resource notifies users officially about something on Sunbird. '
+  termsOfService: https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE
+  contact:
+    email: info@sunbird.org
+  version: "1.0"
+  
+externalDocs:
+ description: "Testcases"
+ url: "https://www.getpostman.com/collections/0d115856f5fb996ad38c"
+servers:
+- url: https://staging.sunbirded.org/api
+paths:
+  /otp/v1/generate:
+    post:
+      tags:
+      - OTP APIs
+      summary: Generate OTP
+      description: |-
+        This API is associated with sending OTP to user on the Sunbird Platform.
+           - The endpoint for **Generate OTP** is `otp/v1/generate`
+           - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      operationId: '{{host}}/otp/v1/generate'
+      parameters:
+      - name: Authorization
+        in: header
+        description: To make use of the API, you require authorization. Raise a request
+          to the administrator for the use of the API. You will receive the authorization
+          key. Specify the key received, here.
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: |-
+          The body refers to the format of the request.
+            - The body contains metadata about the notification to be sent.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/generateOtpRequest'
+        required: true
+      responses:
+        200:
+          description: OK ! Successful operation."Notifying User" operation was successfully
+            executed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContentCreateResponse'
+              example:
+                id: api.otp.generate
+                ver: v1
+                ts: 2020-12-01 10:49:01:948+0000
+                params:
+                  resmsgid: 
+                  msgid: 12eb0ca45ce4c776fab994d120bedd41
+                  err: null
+                  status: success
+                  errmsg: null.
+                responseCode: OK
+                result: 
+                 response: "SUCCESS"
+        400:
+          description: BAD REQUEST. The "Notifying User" operation failed ! The possible
```

Comment:
@sknirmalkar89 This is a copy-paste of the notify API. Please correct.

Reply:
updated